src/spikeinterface/preprocessing/average_across_direction.py

```diff
@@ -132,7 +132,7 @@ def get_traces(self, start_frame, end_frame, channel_indices):
         # now, divide by the number of channels at that position
         traces /= self.n_chans_each_pos
 
-        return traces
+        return traces[:, channel_indices]
```

Comment:
if channel_indices is None then it is adding one dimmension.
could tou test for this ?

Reply:
Oh oops, didn't realize that can be None. Will do. I wasn't sure if I can reopen this PR (for the future -- is that possible?) so I did a new one here: https://github.com/SpikeInterface/spikeinterface/pull/3636